packages/expo-updates/src/Updates.types.ts

```diff
@@ -84,6 +84,54 @@ export type UpdateEvent = {
   message?: string;
 };
 
+/**
+ * An object representing a single log entry from expo-updates logging on the client.
+ */
+export type UpdatesLogEntry = {
+  /**
+   * The time the log was written, in seconds since Jan 1 1970 UTC.
+   */
+  timestamp: number;
+  /**
+   * The log entry message.
+   */
+  message: string;
+  /**
+   * One of the defined code values for expo-updates log entries.
+   */
+  code: UpdatesLogEntryCode;
+  /**
+   * The level or severity of the entry (fatal, error, warn, info, debug, trace)
+   */
+  level: string;
+  /**
+   * If present, the ID of an update associated with this log entry.
+   */
+  updateId?: string;
+  /**
+   * If present, the ID of an asset associated with this log entry.
+   */
+  assetId?: string;
```

Comment:
something to keep in mind for the future is that asset IDs (as stored in SQLite) are essentially meaningless outside of SQLite and are different across devices, so we should use some other unique identifier here -- probably asset hash

Reply:
Thanks for the clarification... I'll change the documentation comment to make it explicit that this will be the unique hash string.

Reply:
Made the doc change.